### DISCUSSION TITLE
✨New Issue Details is Generally Available

### DISCUSSION THREAD

**Discussion by rachrwang** [getsentry/sentry#80801] [Launch Feedback]
## Jan 2025: general availability 
Thank you to everyone who shared feedback during the early adopter release. Based on your input, we’ve made several updates, including improved tags, a more visible JSON button, and better search interactions.

The new issue details experience is now available to all users via the opt-in toggle. We're still iterating and would love your feedback. This experience will become the default in the coming months, so let us know how we can make it even better.

## Nov 2024: early adopter release 

A new issue details experience has started to roll out to early adopter organizations. 

### What's changing and why?
As we built and enriched the issues experience over time, such as directly embedding replays or traces, the overall layout and structure of the page became more complicated and difficult to navigate.

Our updates focus on:

- Improving information hierarchy, particularly around "issue-level" and "event-level" data, so that you can easily understand whether what you're looking at is specific to an event or applies to the overall issue
- Flattening the prior tabs navigation, which includes bringing comments to the forefront 
- Event querying as a top-level action, making it easier to query different timeframes and search specific events
- Enabling expand and collapse of issue sections (including the right side panel), so that your view is optimized based on your preferences

![Screenshot 2024-11-14 at 1 19 55 PM](https://github.com/user-attachments/assets/6198a18c-9aac-4a69-a1fa-fc555fb35309)

### How can I access the new experience?

Within the next few days, those in EA orgs will start seeing this beaker toggle in the upper right of their issue details experience. Clicking on it will take you to the new experience.
![Screenshot 2024-11-14 at 1 21 06 PM](https://github.com/user-attachments/assets/e608c7c0-570d-40da-8fe8-be07dd0cfe33)

It's just as easy to switch back to the old experience, though before you do, please give us feedback! 

![Screenshot 2024-11-14 at 1 24 51 PM](https://github.com/user-attachments/assets/21c5caca-178e-454e-a182-aaa6acf7899d)

### How can I know what changed?
We've put together [a handy guide here](https://sentry.zendesk.com/hc/en-us/articles/30882241712795). If you have any more questions, please feel free to chime in below. 

We'll update this discussion as new updates are rolled out. Thank you all for being a part of our journey!

**Comment by leeandher** [comment]
Quick update here, we've now released the new issue details experience to **all early adopter organizations on Sentry**! If you'd like access, you can toggle your org's EA status in 'Settings > Organization > General Settings'.  As previously described, it's entirely opt-in and can be found by clicking the beaker toggle at the top of the details page. 

The team is actively monitoring and addressing feedback, and we want to make sure this page is a boost to your Sentry experience, so please, make your voice heard!

For self-hosted users, you can access the opt-in for your organization by enabling the `organizations:issue-details-streamline` flag, and please feel free to provide your feedback directly in this thread.

**Comment by ashemedai** [comment]
One thing I am missing a lot in the new view is the removal of the on-hover date/timestamps from the viewers of an issue. This is extremely useful in regression / escalation cases to see if previously people who viewed this are aware of the current situation.

**Reply by leeandher** [reply]
> One thing I am missing a lot in the new view is the removal of the on-hover date/timestamps from the viewers of an issue. This is extremely useful in regression / escalation cases to see if previously people who viewed this are aware of the current situation.

Great catch! Addressing in https://github.com/getsentry/sentry/pull/81584

**Comment by mrduncan** [comment]
We've now released the new issue details experience to **all Sentry organizations**!

As a reminder, it's entirely opt-in at the moment - click the shiny beaker at the top of the issue details page:
![Screenshot 2025-01-23 at 12 25 25 PM](https://github.com/user-attachments/assets/eb0a2abc-f9cb-4fb7-b160-2c8dc7026b12)

We also have additional documentation to help you navigate the new experience: https://sentry.zendesk.com/hc/en-us/articles/30882241712795

Some of my favorite new capabilities:
- The search bar on the issue details page to narrow down to a specific event within an issue
- The event graph lets you pick any time range now instead of previously only supporting 24h/30d
- The trace preview to quickly jump to the trace an event occurred in
- Activity on the right (collapsable!) sidebar makes it so much easier to leave comments for coworkers

We'd love to hear more of your feedback as you check it out!

**Comment by scttcper** [comment]
Check out our tour of the new issue details on YouTube! http://www.youtube.com/watch?v=aGqaxzwGIAs

[![a tour of the new Sentry issue view](http://img.youtube.com/vi/aGqaxzwGIAs/0.jpg)](http://www.youtube.com/watch?v=aGqaxzwGIAs "Pro-level debugging: a tour of the new Sentry issue view")

**Comment by mfn** [comment]
Rarely have I seen design updates updates like this which trigger a positive reception, but I've to say from the very first moment I switched, it just made so much more sense and it's so much easier to grasp the essence of an issue. Very well done!
